tasks/Dockerfile

```diff
@@ -1,6 +1,8 @@
 FROM alpine
 
-RUN apk --update --no-cache add jq curl postgresql-client
+RUN apk --update --no-cache add jq curl postgresql-client py2-pip
```

Comment:
Why py2?

Reply:
Needed to pull `b2` on the following line. Alpine Linux is _super_ minimal